templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -51,47 +51,46 @@ contents:
           [ -e /etc/ssl/etcd/system:etcd-server:${ETCD_DNS_NAME}.crt -a \
             -e /etc/ssl/etcd/system:etcd-server:${ETCD_DNS_NAME}.key ] || \
             kube-client-agent \
-              request \
-                --kubeconfig=/etc/kubernetes/kubeconfig \
-                --orgname=system:etcd-servers \
+              mount \
```

Comment:
do we require a flag here or can we make assumptions based on the presence of the mount?

Reply:
sry do we need to change the command here I should say

Reply:
this is probably fine

Reply:
posting the discussion from slack: `request` feels more like the CSR workflow, so opted for `mount`. With regards to change in manifests for rolling updates and/or installing without CEO, we can use a condition in the template to switch commands as suggested by @hexfusion